.github/workflows/coding-standards.yml

```diff
@@ -94,6 +94,9 @@ jobs:
       - name: Check test suite files for warnings
         run: phpcs tests -q --report=checkstyle | cs2pr
 
+      - name: Ensure version-controlled files are not modified during the tests
+        run: git diff --exit-code
+
   # Runs the JavaScript coding standards checks.
   #
```

Comment:
The new step should be added to the inline documentation at the top of the job in each workflow.